lib/sha256digest-browser.js

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (c) 2022 Digital Bazaar, Inc. All rights reserved.
+ */
+/* eslint-env browser */
+const crypto = self && (self.crypto || self.msCrypto);
```

Comment:
Should this be `globalThis` not `self` now?

Reply:
Changed.